### PR TITLE
Prac 05 code review request

### PR DESCRIPTION
**Pull request by LiautDicksonMaluga**
please check word occurrence to give any tips on formatting the final string properly. @khan-pog

### CODE REVIEW

File: prac_05/emails.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+def main():
+    email_to_name = {}
+    email = input("Email: ")
+    while email != "":
+        name = get_name_from_email(email)
+        confirmation = input("Is your name {}? (Y/n ".format(name))
+        if confirmation.upper() != "Y" and confirmation != "":
```
Comment (khan-pog) [inline prac_05/emails.py:7]: Did you have a reason for making != "Y", and != "". instead of = "n".

File: prac_05/state_names.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+print(CODE_TO_NAME)
+
+# state_code = input("Enter short state: ")
```
Comment (khan-pog) [inline prac_05/state_names.py:11]: Is there a reason why you left this comment in the final copy?

File: prac_05/word_occurrences.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+for word in words:
+    if word in words_to_count:
+        words_to_count[word] += 1
+    else:
+        words_to_count[word] = 1
+
+words = list(words_to_count.keys())
+words.sort()
+
+longest_word = max((len(word) for word in words))
```
Comment (khan-pog) [inline prac_05/word_occurrences.py:14]: I like this code. Smart.

**Review by khan-pog — COMMENTED** [review]
Keen to hear back from you on those question.

File: prac_05/state_names.py (hunk 2 above)
Comment (LiautDicksonMaluga) [reply prac_05/state_names.py:11]: I do not have a reason, it was part of the starting code however wasn't the final result asked for. I just didn't know whether to remove it or not.

File: prac_05/word_occurrences.py (hunk 3 above)
Comment (LiautDicksonMaluga) [reply prac_05/word_occurrences.py:14]: I was actually struggling with this code so I don't know if you are joking or not, the code does not work as intended and i can't get the right formatting at the end.

File: prac_05/emails.py (hunk 1 above)
Comment (LiautDicksonMaluga) [reply prac_05/emails.py:7]: No reason, your way is actually better thank you

**Pull request merged**